libs/dh/metering-point/shared/ui-util/src/lib/dh-yes-no.pipe.ts

```diff
@@ -14,20 +14,23 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { NgModule, Pipe, PipeTransform } from '@angular/core';
-import { TranslocoModule, TranslocoService } from '@ngneat/transloco';
+import { Pipe, PipeTransform, inject } from '@angular/core';
+import { TranslocoService } from '@ngneat/transloco';
 
 export type TValue = string | boolean | undefined | null;
 export const pipeName = 'dhYesNo';
 
 @Pipe({
   name: pipeName,
```

Comment:
nitpick: now that the test is not using Spectacular anymore, the `TValue` and `pipeName` don't have be exported. The `pipeName` can directly go in the pipe's name instead.